integration-testing/rnode_testing/docker.py

```diff
@@ -2,9 +2,11 @@
 from contextlib import contextmanager
 import rnode_testing.random
 
+from docker.client import DockerClient
+from typing import Generator
 
 @contextmanager
-def docker_network(docker_client):
+def docker_network(docker_client: DockerClient) -> Generator[str, None, None]:
```

Comment:
`Generator` is too generic.  There must be some more specialized type that doesn't indicate a series of values is produces but just one.

Reply:
Another return type for this one is `Iterator[str]`

Reply:
For a function which would `yield`, the return type is either `Iterator[yield_object_type]` or `Generator[yield_object_type, None, None]`

More information for the generator should see this https://docs.python.org/3/library/typing.html#typing.Generator